plugins/infinispan90/pom.xml

```diff
@@ -124,6 +124,17 @@
          <optional>true</optional>
       </dependency>
 
+      <dependency>
+         <groupId>org.infinispan</groupId>
+         <artifactId>infinispan-marshaller-kryo</artifactId>
+         <version>${version.infinispan}</version>
```

Comment:
My Maven build can't find the artifacts (Kryo and Protostuff) because I have 9.0.0-SNAPSHOT in version.infinispan. We should probably use an independent version for the Kryo and Protostuff libs. I can see that Nexus contains only 9.0.0.Beta1